dolphinscheduler-api/src/main/resources/application-api.properties

```diff
@@ -56,6 +56,9 @@ security.authentication.type=PASSWORD
 #traffic.control.default.tenant.qps.rate=10
 #traffic.control.customize.tenant.qps.rate={'tenant1':11,'tenant2':20}
 
+# Audit log control
+#audit.control.global.switch=false
```

Comment:
The switch can set true, otherwise the front page will display white. Or you have a better way to let the front end don't show audit tab